features/user-associated-with-multiple-accounts.md

```diff
@@ -0,0 +1,26 @@
+# User associated with multiple accounts
```

Comment:
How about "Multi-account user association"? We are already using "multi-account users" in a sub heading below. @abhinavr4 Thoughts?

Let's change the link and file name also.

Reply:
@niksh @abhinavr4  then changing it to "Multi-account user association".

Reply:
done